doc/api/async_hooks.md

```diff
@@ -682,6 +682,85 @@ never be called.
 * Returns: {number} The same `triggerAsyncId` that is passed to the
 `AsyncResource` constructor.
 
+## Class: AsyncContext
```

Comment:
node-addon-api also has a class [`Napi::AsyncContext`](https://github.com/nodejs/node-addon-api/blob/master/doc/async_context.md), so this may be confusing.

Reply:
Good point! I am totally ok with changing name to something like `AsyncStorage` or `AsyncLocal(s)` if there is a consencus on something.

Reply:
I would use `AsyncLocal` only if it is similar to the [.NET `AsyncLocal`](https://docs.microsoft.com/en-us/dotnet/api/system.threading.asynclocal-1?view=netframework-4.8) otherwise just other users get confused.

Reply:
I don't have any strong opinions here. AsyncLocal works for me.

However, let's keep AsyncContext until a decision is made regarding which PR will ber merged. It is handy to refer at this PR as AsyncContext and the other one as AsyncLocal.